packages/react-devtools-shell/src/app/index.js

```diff
@@ -28,6 +28,7 @@ ignoreErrors([
   'Warning: Legacy context API',
   'Warning: Unsafe lifecycle methods',
   'Warning: %s is deprecated in StrictMode.', // findDOMNode
+  'Warning: ReactDOM.render is no longer supported in React 18',
```

Comment:
It's okay to ignore this in the DevTools test harness. We _intentionally_ use legacy `render` so that we can integration-test it. Once it's (eventually) removed, we'll remove it too in DevTools. (Or we'll add a legacy version of React to the test harness.)